lib/harvesting/models/base.rb

```diff
@@ -2,11 +2,11 @@ module Harvesting
   module Models
     class Base
       attr_accessor :attributes
-      attr_reader :client
+      attr_reader :harvest_client
```

Comment:
@mscottford Cool! I was a bit worried about having two things called "client" in this library. This makes it easier to read. 👍